infra/examples/aws-google-workspace/main.tf

```
@@ -187,3 +187,60 @@ module "worklytics-psoxy-connection" {
   display_name       = "${each.value.display_name} via Psoxy"
 }
 
+
+# BEGIN LONG ACCESS AUTH CONNECTORS
+
+locals {
+  oauth_long_access_connectors = {
+    slack = {
+      deploy = true
+      function_name = "psoxy-slack-discovery-api"
+      source_kind = "slack"
+      example_api_calls = []
+    },
+    zoom = {
+      deploy = true
+      function_name = "psoxy-zoom"
+      source_kind = "zoom"
+      example_api_calls = []
+    }
+  }
+}
+
+# Create secret (later filled by customer)
+resource "aws_ssm_parameter" "long-access-token-secret" {
+  for_each = {
+  for k, v in local.oauth_long_access_connectors:
+  k => v if v.deploy
+  }
+  name        = "PSOXY_ACCESS_TOKEN_${each.value.function_name}"
+  type        = "SecureString"
+  description = "The long lived token for ${each.value.function_name}"
+  value       = sensitive("anything")
```

Or if you prefer to apply the suggestion of sensitive variables:

```suggestion
  value       = var.access_token
```